restcomm/restcomm.extension.api/src/main/java/org/restcomm/connect/extension/api/SessionExtensionResponse.java

```diff
@@ -0,0 +1,11 @@
+package org.restcomm.connect.extension.api;
+import org.apache.commons.configuration.Configuration;
+public class SessionExtensionResponse extends ExtensionResponse {

```

Comment:
@gvagenas added
I think the design of Extensions, the exposed API, ExtensionsResponse is something we still need to discuss.
Initially i think of `SessionResponse` as Sms agnostic, for me i guess a running Call might be defined a 'Session'